specs/commandline/sign.md

```diff
@@ -27,58 +41,79 @@ Flags:
 
 ## Usage
 
-### Sign a container image
+### Sign an OCI artifact stored in a registry using a remote key
 
 ```shell
-# Add a key which uses a local private key and certificate, and make it a default signing key
-notation key add --default --name <key_name> <key_path> <cert_path>
-
-# Or change the default signing key to an existing signing key
-notation key update --default <key_name>
+# Prerequisites: 
+# - A compliant signing plugin is installed in notation. See notation plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
```

Comment:
```suggestion
# - A signing plugin is installed, dee [plugin documentation](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
```

Reply:
It's in the comment, so the `[]()` format doesn't work.